src/components/menu/style.css

```diff
@@ -0,0 +1,72 @@
+#menu__toggle {
```

Comment:
I think we've already discuss this. It's not a good practice to use ids to style components. You should use classes instead.

Reply:
Yes, I had forgotten about our talk on this subject. I will make this change in a moment.